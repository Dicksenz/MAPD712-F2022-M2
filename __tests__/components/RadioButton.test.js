import React from "react";
import renderer from "react-test-renderer";
import RadioButton from "../../components/RadioButton";

test("Check if RadioButton component renders correctly", () => {
  const tree = renderer.create(<RadioButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
