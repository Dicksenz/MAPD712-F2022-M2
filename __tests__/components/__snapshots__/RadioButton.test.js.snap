// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if RadioButton component renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#ACACAC",
      "borderRadius": 10,
      "borderWidth": 1,
      "height": 20,
      "justifyContent": "center",
      "marginHorizontal": 10,
      "opacity": 1,
      "width": 20,
    }
  }
>
  <View />
</View>
`;
